src/test/java/core/basesyntax/service/impl/ReadCsvImplTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.ReadCsvService;
+import exception.FileException;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReadCsvImplTest {
+    private static final String PATH_TO_FILE = "src/main/resources/balance.csv";
+    private static final String WRONG_PATH_TO_FILE = "src/main/service/balance.csv";
+    private static final String PATH_TO_EMPTY_FILE = "src/main/resources/empty.csv";
+    private ReadCsvService readCsvService;
+    private List<String> expectedFileContent = new ArrayList<>();
+
+    @BeforeEach
+    void setUp() {
+        readCsvService = new ReadCsvImpl();
+        expectedFileContent = readCsvService.readFromFile(PATH_TO_FILE);
+    }
+
+    @Test
+    void read_Ok() {
+        List<String> actual = readCsvService.readFromFile(PATH_TO_FILE);
+        assertEquals(expectedFileContent, actual);
+    }
+
+    @Test
+    void readNull_NotOk() {
+        assertNull(null, "File don`t have any data");
+    }
+
+    @Test
+    void readFromEmptyFile_NotOk() {
```

Comment:
Method is called "NotOk", but seems that everything is OK)

Reply:
Now, not all is Ok))    
@Test
    void readFromEmptyFile_NotOk() {
        final List<String> actual = readCsvService.readFromFile(PATH_TO_EMPTY_FILE);
        final List<String> expected = readCsvService.readFromFile(PATH_TO_FILE);
        assertNotEquals(actual, expected);
    }